backend/entityservice/async_worker.py

```diff
@@ -21,31 +21,15 @@
 celery.conf.worker_max_tasks_per_child = Config.CELERYD_MAX_TASKS_PER_CHILD
 
 
-structlog.configure(
-    processors=[
-        structlog.stdlib.add_logger_name,
-        structlog.stdlib.add_log_level,
-        structlog.stdlib.PositionalArgumentsFormatter(),
-        structlog.processors.StackInfoRenderer(),
-        structlog.processors.format_exc_info,
-        structlog.dev.ConsoleRenderer()
-    ],
-    context_class=structlog.threadlocal.wrap_dict(dict),
-    logger_factory=structlog.stdlib.LoggerFactory(),
-)
-
-# Set logging level of other python libraries that we use
-logging.getLogger('celery').setLevel(logging.WARNING)
-logging.getLogger('jaeger_tracing').setLevel(logging.WARNING)
-
 # Set up our logging
-logger = structlog.wrap_logger(logging.getLogger('celery.es'))
+setup_structlog()
+logger = structlog.wrap_logger(logging.getLogger('entityservice.tasks'))
 if Config.DEBUG:
-    logging.getLogger('celery.es').setLevel(logging.DEBUG)
-    logging.getLogger('celery').setLevel(logging.INFO)
+    logging.getLogger('entityservice.tasks').setLevel(logging.DEBUG)
```

Comment:
you also define a log level in the logging.yaml file. How does that relate to `Config.DEBUG`? And which value is set eventually?

Reply:
`Config.DEBUG` is an old setting that toggles between default and verbose logging based on an environment variable. 

I think I should remove it entirely as the same could be achieved by shipping with a `verbose_logging.yaml` and a `default_logging.yaml`